src/main/resources/beam-template.conf

```diff
@@ -442,6 +442,9 @@ beam.physsim.pickUpDropOffAnalysis.secondsFromPickUpPropOffToAffectTravelTime =
 beam.physsim.pickUpDropOffAnalysis.additionalTravelTimeMultiplier = "double | 1.0"
 
 # JDEQSim
+beam.physsim.jdeqsim.parameters.alpha = "double | 1.0"
+beam.physsim.jdeqsim.parameters.beta = "double | 2.0"
```

Comment:
we need either self-explanatory names for parameters or a good comments about them, 
also a good idea is to put a comment with a range of possible values

Reply:
Good point, thank you! Will add comments after we figure out the range of possible values.

Reply:
abadonded this version, instead we have alpha and beta for different link categories now.